neqo-common/src/qlog.rs

```diff
@@ -36,6 +58,7 @@ impl Qlog {
     /// # Errors
     ///
     /// Will return `qlog::Error` if it cannot write to the new file.
+    #[cfg(feature = "qlog")]
```

Comment:
Again, I would not feature guard the entire function, unless it is to define an alternative function implementation that returns a neutered value (without `inner`).